### ISSUE TITLE
VCL: Memo editor gutter tweak needed on dark theme

### ISSUE THREAD

**Issue by darianmiller** [ideasawakened/DelphiKB#21]
`2020.01.25` reported to vendor
procedure procedure TAdvMemo.AdvSyntaxMemoGutterDraw

Changed color of the dots displayed in the Gutter if displaying line numbers every Xth line

One liner from (line 16997)  
`ACanvas.Pen.Color := FGutter.Font.Color;`  
to:  
`ACanvas.Pen.Color := FGutter.FLineNumberTextColor;`

End up with:
````
      if ((AcLineNo + FGutter.FLineNumberStart) mod FGutter.LineNumberAt <> 0) and (AcLineNo <> CurY) then
      begin
        ACanvas.Pen.Color := FGutter.FLineNumberTextColor;
        ACanvas.Pen.Width := 1;
        ACanvas.MoveTo(Right - 18, Top + 4);
        ACanvas.LineTo(Right - 16, Top + 4);
      end;
````

**Comment by darianmiller** [comment]
`2020.01.26` vendor response
From: TMS software <info@tmssoftware.com> 
Sent: Sunday, January 26, 2020 2:31 AM
To: darian@ideasawakened.com
Subject: RE: Support: Miller Darian : TAdvMemo : TAdvMemo gutter color tweak

Dear Mr. Miller,

You’re correct about this. We’ve applied this suggested improvement.

Thanks again for your valuable inputs!


Kind regards,
Bruno Fierens

**Comment by darianmiller** [comment]
`2020.01.29` Reported fixed in v10.1.6.2
